docs/testing-with-applitools-eyes.md

```diff
@@ -4,10 +4,10 @@ Applitools Eyes is a service that lets us upload and compare screenshots of our
 
 ## Getting an account
 
-Code.org employees as of this writing should receive login details for a shared account. If not, contact Anthony about getting an account.
+As of November 2021, you can use your GitHub credentials or your Google Code.org credentials to get an account.
```

Comment:
This is how I did it when @sureshc paired with me to get the account setup when I was starting

Reply:
I’d prefer `Consult the “Getting Started as a Developer at Code.org” document for details on how to gain access to SauceLabs`. Probably fine to link to that Google Doc https://docs.google.com/document/d/158MiIxJ_1JokzUkw43Wf2bFONPycYoCeXtKEU-ww0Xw/edit

Reply:
Good call––just updated